packages/paste-core/components/tabs/src/TabsContext.tsx

```diff
@@ -1,11 +1,12 @@
 import * as React from 'react';
-import {TabPrimitiveState} from '@twilio-paste/tabs-primitive';
-import {Variants} from './types';
+import type {TabPrimitiveState} from '@twilio-paste/tabs-primitive';
+import type {Variants} from './types';
 
-interface TabState extends TabPrimitiveState {
+interface TabState extends Omit<TabPrimitiveState, 'orientation'> {
   variant?: Variants;
+  orientation: 'horizontal' | 'vertical';
```

Comment:
Just using the type value of `Orientation` since that specific type is not exported from reakit this is the simplest way to type this.